api/src/main/java/com/debugmafia/clueless/state/Game.java

```diff
@@ -28,7 +28,6 @@
   public Game(Set<Player> players) {
     this.board = new Board();
     this.gameState = GameState.IN_PROGRESS;
-    this.players = new ArrayList<>(players);
```

Comment:
I saw that you moved this line, which is okay and is working fine from what I can tell. Just noting here that I had put this line here originally because the app was crashing when a new Game was created. This was because the `sortAndAssignPlayerList()` method was trying to use `this.players` before it had been assigned, so assigning it here fixed that issue.

Reply:
Since this is now being done within sortAndAssignPlayerList, I think we're okay. I had no issues when I was running last night.